llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -4857,9 +4857,11 @@ MachineBasicBlock *SITargetLowering::EmitInstrWithCustomInserter(
     bool IsAdd = (MI.getOpcode() == AMDGPU::S_ADD_U64_PSEUDO);
     if (Subtarget->hasScalarAddSub64()) {
       unsigned Opc = IsAdd ? AMDGPU::S_ADD_U64 : AMDGPU::S_SUB_U64;
+      // clang-format off
       BuildMI(*BB, MI, DL, TII->get(Opc), Dest.getReg())
-          .addReg(Src0.getReg())
-          .addReg(Src1.getReg());
+        .add(Src0)
+        .add(Src1);
+      // clang-format on
```

Comment:
Adding these comments is worse than putting up with clang-format's preferred indentation.

Reply:
Drop the comments and ignore the error. I really wish clang-format would just ignore anything involving BuildMI